compat/src/index.d.ts

```diff
@@ -55,6 +56,8 @@ declare namespace React {
 	export import cloneElement = preact.cloneElement;
 	export import ComponentProps = preact.ComponentProps;
 	export import ReactNode = preact.ComponentChild;
+	export import ReactElement = preact.VNode;
```

Comment:
`VNode` seemed to be the closest thing in preact to ReactElement from what I could gather, but this might be wrong.

Reply:
Seems right, `VNode` looks to cover all the same properties (and a couple more).